lib/ansible/modules/network/nxos/nxos_ospf_vrf.py

```diff
@@ -90,6 +90,13 @@
     description:
       - Specifies the reference bandwidth used to assign OSPF cost.
         Valid values are an integer, in Mbps, or the keyword 'default'.
+  bfd:
+    description:
+      - Enables BFD on all OSPF interfaces.
+      - "Dependency: 'feature bfd'"
+    version_added: "2.9"
+    type: bool
```

Comment:
Would you please change this to type str which choices [disable, enable]? This would make bfd type consistent across all the modules.

Reply:
@trishnaguha I considered changing from bool on these as well but held off because these really are bools while the interface modules have the tri-state values. But on second look I agree that it's cleaner to make these consistent. I'll make the changes. ETA: 6/18

Reply:
Thanks!

Reply:
@trishnaguha Changes made. Retested on all regression platforms.